iml-services/iml-ntp/src/main.rs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2019 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use iml_ntp::{
+    db::{add_alert, get_active_alert_for_fqdn, get_managed_host_items, set_alert_inactive},
+};
+
+use futures::stream::{StreamExt, TryStreamExt};
+use iml_postgres::connect;
+use iml_wire_types::{ntp::{TimeSync, TimeStatus}, Fqdn};
+use iml_service_queue::service_queue::consume_data;
+use tracing_subscriber::{fmt::Subscriber, EnvFilter};
+
+pub fn valid_state(mh_state: String) -> Option<String> {
+    ["monitored", "managed", "working"]
+        .iter()
```

Comment:
If you use into_iter here instead, I think you can avoid the to_string at the end.

Reply:
I moved the `is_some` inside the function so it can return a bool, which will be nicer to work with.